src/cdk/tree/tree.spec.ts

```diff
@@ -26,26 +27,215 @@ describe('CdkTree', () => {
       imports: [CdkTreeModule],
       declarations: [
         SimpleCdkTreeApp,
-        // TODO(tinayuangao): Add more test cases with the cdk-tree
-        //   DynamicDataSourceCdkTreeApp,
-        //   NodeContextCdkTreeApp,
-        //   WhenNodeCdkTreeApp
+        NestedCdkTreeApp,
+        CdkTreeAppWithTrigger,
+        NestedCdkTreeAppWithTrigger,
+        WhenNodeCdkTreeApp,
+        WhenNodeNestedCdkTreeApp,
```

Comment:
Can you break up these into separate describe blocks (similar to the mat-select tests)? Organized this way, Angular will compile each test component for every `it`, which pretty dramatically slows down the tests (this is why IE11 has been timing out recently). This will also resolve the `ComponentFixture` being `any`

Reply:
Done. Please take a look. Thanks!